scripts/west_commands/runners/nrfjprog.py

```diff
@@ -113,8 +114,46 @@ def do_run(self, command, **kwargs):
                              format(self.hex_) +
                              'Try enabling CONFIG_BUILD_OUTPUT_HEX.')
 
+        if self.family == 'NRF53':
```

Comment:
Do you need to update the test cases that are specific to the nrf53 also? It looks like the answer is yes, since the flash commands may change.

See scripts/west_commands/README.txt for information on running tests. The test cases for nrfjprog are in scripts/west_commands/tests/test_nrfjprog.py. You will need to learn a bit about py.test -- let me know if you want to work together on this.